### ISSUE TITLE
Broken Tooltip descriptions

### ISSUE THREAD

**Issue by Miskler** [godotengine/godot#59950]
### Godot version

3.4.4.stable

### System information

Windows 10, Gles 3, 8 GB RAM, Intel i5-3470 3.(2-6)GHz

### Issue description

https://user-images.githubusercontent.com/69055489/162033168-6aa2f715-0075-4e61-b48f-dd89414d7bb0.mp4

Any Tooltip only show a description after the cursor exits the window, provided that the last place where the cursor was registered had a Tooltip

### Steps to reproduce

1) Create a new project,
2) Create a scene Control,
3) Give Control any description,
4) Run the scene.

### Minimal reproduction project

[d.zip](https://github.com/godotengine/godot/files/8429141/d.zip)

**Comment by Miskler** [comment]
I looked here, if the game is exported, then everything will be fine. The bug is only in the editor.

**Comment by KoBeWi** [comment]
Duplicate of #46737

**Comment by Miskler** [comment]
The problem appeared relatively recently, before that everything was fine.
The only thing I did between the failure and the presence of an error was
to update Windows. This problem appeared after the update.It is present on
all versions of the engine that I tested (3.2.3 - 3.4.4). In the exported
project in debug mode and without it, there is no problem, it is only in
the editor.

ср, 13 апр. 2022 г., 00:42 Tomek ***@***.***>:
